pkg/sdn/plugin/node.go

```diff
@@ -204,9 +202,23 @@ func (node *OsdnNode) Start() error {
 		}
 	}
 
-	// Wait for kubelet to init the plugin so we get a knetwork.Host
-	log.V(5).Infof("Waiting for kubelet network plugin initialization")
-	<-node.kubeletInitReady
+	log.V(5).Infof("Waiting for kubelet and network plugin initialization")
+	err = kwait.ExponentialBackoff(
```

Comment:
I was under the impression from other PRs that @smarterclayton wants all polling to use exponential backoff.

In this case the only way the condition wouldn't be true almost immediately is if the host is egregiously overloaded, in which case backoff seems good to keep it from getting even more overloaded.

[PR updated for the other comments, but I didn't change anything here yet...]